src/promptflow/tests/sdk_cli_azure_test/e2etests/test_flow_operations.py

```diff
@@ -73,3 +74,11 @@ def test_list_flows(self, pf):
         for flow in flows:
             print(json.dumps(flow._to_dict(), indent=4))
         assert len(flows) == 3
+
+    def test_eager_flow_creation(self, pf):
+        flow_source = eager_flow_test_dir / "simple_with_yaml"
```

Comment:
Can move to unit test?

Reply:
And may also need to update recording for `test_create_flow` since there is code change

Reply:
Moved to UT, seems the newly added logic won't affect recording, the CI's passing.